app/models/daos/slick/UserDAOSlick.scala

```diff
@@ -190,9 +190,9 @@ object UserDAOSlick {
         "(label_validation.end_timestamp AT TIME ZONE 'US/Pacific')::date = (NOW() AT TIME ZONE 'US/Pacific')::date",
         "(audit_task.task_end AT TIME ZONE 'US/Pacific')::date = (NOW() AT TIME ZONE 'US/Pacific')::date"
       )
-      case "yesterday" => (
-        "(label_validation.end_timestamp AT TIME ZONE 'US/Pacific')::date = (now() AT TIME ZONE 'US/Pacific')::date - interval '1' day",
-        "(audit_task.task_end AT TIME ZONE 'US/Pacific')::date = (now() AT TIME ZONE 'US/Pacific')::date - interval '1' day"
+      case "past week" => (
+        "(label_validation.end_timestamp AT TIME ZONE 'US/Pacific') > (now() AT TIME ZONE 'US/Pacific') - interval '168 hours'",
+        "(audit_task.task_end AT TIME ZONE 'US/Pacific') > (now() AT TIME ZONE 'US/Pacific') - interval '168 hours'"
       )
       case "week" => (
         "(label_validation.end_timestamp AT TIME ZONE 'US/Pacific')::date > DATE_SUB(NOW() AT TIME ZONE 'US/Pacific', INTERVAL 1 WEEK)",
```

Comment:
Can you remove the "week" and "month" options that are already here? I think that Jon added them so that they could theoretically be used in the future, but `DATE_SUB` is actually a MySQL function anyway and won't work in PostgreSQL. You can change yours from "past week" to just "week" then :) And update the comments and stuff too, please!